tests/prague/eip7692_eof_v1/eip7069_extcall/test_address_space_extension.py

```diff
@@ -76,21 +76,9 @@ def test_address_space_extension(
     if ase_address and target_address[0] == b"00":
         raise ValueError("Test instrumentation requires target addresses trim leading zeros")
 
-    match target_opcode:
-        case Op.CALL | Op.CALLCODE:
-            call_suffix = [0, 0, 0, 0, 0]
-            ase_ready_opcode = False
-        case Op.DELEGATECALL | Op.STATICCALL:
-            call_suffix = [0, 0, 0, 0]
-            ase_ready_opcode = False
-        case Op.EXTCALL:
-            call_suffix = [0, 0, 0]
-            ase_ready_opcode = True
-        case Op.EXTDELEGATECALL | Op.EXTSTATICCALL:
-            call_suffix = [0, 0]
-            ase_ready_opcode = True
-        case _:
-            raise ValueError("Unexpected opcode ", target_opcode)
+    ase_ready_opcode = (
```

Comment:
Interestingly, if I left this expression as a (simplified) `match-case` statement, `mypy` complained about the type of `target_opcode` not being callable in the `target_opcode(address=Op.CALLDATALOAD(0))` below. I've searched for the reason and also tried upgrading mypy to 1.11 - to no avail. Have I missed sth or maybe it's worth to produce a reproducible example and report?